docs/deploy.md

```diff
@@ -0,0 +1,148 @@
+# Deploying M3DB
+
+## Deploying a single node
+
+Deploying a single-node cluster is a great way to experiment with M3DB and get a feel for what it
+has to offer. Our Docker image by default configures a single M3DB instance with an embedded etcd
+server (used for runtime configuration flags and topology management). This will allow you to read
+and write timeseries data and change the topology of the instance.
+
+To begin, first start up a Docker container with port `7201` (used to manage the cluster topology)
+and port `9003` (used to read and write metrics) exposed. We recommend you create a persistent data
+directory on your host for durability:
+
+```
+docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3db/m3db:latest
```

Comment:
Don't want to expose the debug port?

Reply:
Yeah good point so people can scrape metrics

Reply:
Also curious if docker compose will be easier to run as oppose to a multi flag incantation?

Reply:
I think we can mention a compose file is available, but that this should be the primary example in the docs a few reasons:

1. `docker run` is the lowest common denominator: some people may have `docker` but not `docker-compose`. Some distributions package them together, but `docker-compose` has its own entire [install](https://docs.docker.com/compose/install/#install-compose) page so I consider it separate. [InfluxDB](https://hub.docker.com/_/influxdb/) does the same, and their flags seem more unwieldy than ours. So does [prometheus](https://github.com/prometheus/prometheus#docker-images) but their flags are pretty simple.

2. `docker run` can be ran from anywhere `docker` is installed, `docker-compose` would require the additional step of downloading the yaml config.

3. I'd rather not abstract away things like persisting the data directory and exposing critical ports from the user. I think it's important to know why certain ports are necessary and see how we persist data. If it's explicit here then when they go to run it on in a more prod-like environment they'll know what flags they need.